src/design-system/components/Badge/Badge.css

```diff
@@ -0,0 +1,7 @@
+.badge {
+  padding: 4px;
```

Comment:
Looking forward hearing about your planned exploration around the css vars for spacing and sizes!